README.md

```diff
@@ -90,6 +90,22 @@ OobUpdate.sh is a program for updating various component firmware of BlueField D
     New BIOS Firmware Version:
             ATF--v2.2(release):4.7.0-25-g5569834, UEFI--4.7.0-42-g13081ae
 
+### Update bundle
```

Comment:
FW Bundle

Reply:
Let's add simple word likes: "Bluefield-firmware-bundle - including only firmware components ATF, UEFI, BMC, CEC and NIC Firmware"